common/header.html

```diff
@@ -203,6 +203,6 @@
   });
   
   </script>
-  <script type='text/x-handlebars' data-template-name='/connectors/above-site-header/brand-header'>
+  <script type='text/x-handlebars' data-template-name="/connectors/{{theme-setting 'location'}}-site-header/brand-header">

```

Comment:
Did you test this? You cannot put handlebars in regular HTML, only inside handlebars.